fvtest/gctest/StartupManagerTestExample.cpp

```diff
@@ -84,7 +84,12 @@ MM_StartupManagerTestExample::parseLanguageOptions(MM_GCExtensionsBase *extensio
 				} else if (0 == strcmp(attr.name(), "gcthreadCount")) {
 					/* TODO: support multi-thread GC*/
 				} else if (0 == strcmp(attr.name(), "GCPolicy")) {
+#if defined(OMR_GC_MODRON_SCAVENGER)
 					extensions->scavengerEnabled = (0 == j9_cmdla_stricmp(attr.value(), "gencon"));
+#if defined(OMR_GC_MODRON_CONCURRENT_MARK)
+					extensions->concurrentMark = extensions->scavengerEnabled;
```

Comment:
There is no way to enable concurrent global collections without enabling a generational collector?